lib/public/views/lhcPeriods/format/formatNameWithAssociationsLinks.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { frontLink } from '../../../components/common/navigation/frontLink.js';
+import { h } from '/js/src/index.js';
+
+/**
+ * Format Lhc Periods name with links to associations
+ * @param {string} name lhcPeriodName
+ * @param {LhcPeriod} periodParameters lhc period parameters
+ * @return {Component} name cell display
+ */
+export const formatNameWithAssociationsLinks = (name, { id, runsCount, dataPassesCount }) =>
```

Comment:
Again, please use not ambiguous function names. I'm ok if this function is internal to a module, but if you export it, it must be fully explicit. Here, the LHC period must appear!

Some says that a good function should not have documentation, it should be explicit enough. In your doc you have `Format Lhc Periods name with links to associations`, then your function should be `formatLhcPeriodNameWithLinksToAssociations`.

Reply:
renamed